symbols.xml

```diff
@@ -1063,7 +1063,7 @@
         <vtable-address name='viewscreen_worldst' value='0x141172fe0'/>
         <vtable-address name='viewscreenst' value='0x141173018'/>
         <vtable-address name='widget_button' value='0x1412b7da8'/>
-        <vtable-address name='widget_container' value='0x141173340'/>
+        <vtable-address name='widgets::container' value='0x141173340'/>
         <vtable-address name='widget_tabs' value='0x141180b80'/>
         <vtable-address name='widget_textbox' value='0x1412b7e08'/>
         <vtable-address name='widget_widget' value='0x1412b7d80'/>
```

Comment:
all of the `widget_xxx` names should actually be `xxx::widgets`

i'll push an update to `scan_vtables.rb` asap